### PR TITLE
Dynamic Terrain doc typos

### PR DESCRIPTION
**Pull request by ThomasBurgess2000**
Fixed some typos and rephrased some sentences for clarity.

### CODE REVIEW

File: content/communityExtensions/dynamicTerrains.md [hunk 1]
```diff
@@ -10,26 +10,25 @@ video-content:
...
+The dynamic terrain is a standard BJS mesh, specifically a ribbon. It's linked to a camera and moves with it along the World X and Z axes. It's passed a set of 3D coordinates _(x, y, z)_ in the World. This set of data is called the map. According to the current camera position in the World, the dynamic terrain morphs to depict the map at this location.
 
 ## Installation
 
-Just download the javascript file `dynamicTerrain.js` (or, recommended, the minified version `dynamicTerrain.min.js`) from theBabylon.js [extension repository](https://github.com/BabylonJS/Extensions) folder `DynamicTerrain/dist` : https://github.com/BabylonJS/Extensions/tree/master/DynamicTerrain/dist
+Just download the JavaScript file `dynamicTerrain.js` (or, recommended, the minified version `dynamicTerrain.min.js`) from theBabylon.js [extension repository](https://github.com/BabylonJS/Extensions) folder `DynamicTerrain/dist`: https://github.com/BabylonJS/Extensions/tree/master/DynamicTerrain/dist
```
Comment (sebavan) [inline content/communityExtensions/dynamicTerrains.md]: missing space between the and Babylon.js

**Review by sebavan — COMMENTED** [review]
thanks a lot, let see if @PatrickRyanMS can do a quick pass as I am not English speaker :-)

File: content/communityExtensions/dynamicTerrains.md (hunk 1 above)
Comment (ThomasBurgess2000) [reply content/communityExtensions/dynamicTerrains.md]: Thank you, pushed some more fixes.

File: content/communityExtensions/dynamicTerrains.md [hunk 2]
```diff
@@ -10,26 +10,25 @@ video-content:
...
 
-Then in your code, declare this script in a html tag **after** the script tag declaring Babylon.js :
+Then in your code, declare this script in an html tag **after** the script tag declaring Babylon.js:
 
 ```html
 <script src="babylon.js"></script>
 <script src="dynamicTerrain.min.js"></script>
 ```
 
+It has also been converted to an [ES6 compatible TypeScript module](https://gist.github.com/danielstenson/07bf127136cd7730dfcfedb4e97a9649).
```
Comment (RaananW) [inline content/communityExtensions/dynamicTerrains.md:26]: I will appreciate the code in the docs and not in a gist of an account we don't have control of (i know it is yours :-) )

File: content/communityExtensions/dynamicTerrains.md [hunk 3]
```diff
@@ -10,26 +10,25 @@ video-content:
 
 The Dynamic Terrain is an extension that provides a way to display a large terrain dynamically morphed from a map of 3D data.
 
-![dynamicTerrain](/img/extensions/dynamicTexture.jpg)
```
Comment (RaananW) [inline content/communityExtensions/dynamicTerrains.md:13]: Any reason the image was removed?

**Review by RaananW — COMMENTED** [review]
Thanks a lot! my initial review, before diving in and reading everything.

File: content/communityExtensions/dynamicTerrains.md (hunk 2 above)
Comment (ThomasBurgess2000) [reply content/communityExtensions/dynamicTerrains.md:26]: It's actually not mine, I just found it in this thread and found it useful: https://forum.babylonjs.com/t/need-help-with-including-dynamic-terrain-extension-typeerror-dynamicterrain-is-not-a-constructor/1576/20?u=thomas

Where is the proper place for it in the docs? And can I paste it there without the author's permission?

File: content/communityExtensions/dynamicTerrains.md (hunk 3 above)
Comment (ThomasBurgess2000) [reply content/communityExtensions/dynamicTerrains.md:13]: It was a dead link

File: content/communityExtensions/dynamicTerrains.md (hunk 2 above)
Comment (RaananW) [reply content/communityExtensions/dynamicTerrains.md:26]: I see. sorry, i thought you were the author. Let's keep the URL then.

**Review by RaananW — APPROVED** [review]

**Pull request merged**